caffe2/serialize/inline_container.cc

```diff
@@ -206,7 +185,7 @@ void PyTorchStreamReader::init() {
   std::string version(
       static_cast<const char*>(version_ptr.get()), version_size);
   try {
-    version_ = std::stoull(version);
+    version_ = std::stoll(version);
```

Comment:
Why is this better?

Reply:
Because version_ of PyTorchStreamReader is int64_t, which is inconsistent to version_ in other classes. Now it's changed to uint64_t. The change is backward compatible.